From ea14ff02b0ed6438e2711e278ad1fe9cb1be3384 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Mon, 13 Jan 2014 22:10:39 +0000
Subject: [PATCH] fish: Add test-file-attrs.sh to EXTRA_DIST.

This fixes commit efb5f1841524ec75d765368b447270b8a3aff8ac.

(cherry picked from commit aba64a0327094b6a82686411c4dfa7a30d64ecee)
---
 fish/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fish/Makefile.am b/fish/Makefile.am
index fb0e99e..54ba875 100644
--- a/fish/Makefile.am
+++ b/fish/Makefile.am
@@ -306,6 +306,7 @@ EXTRA_DIST += \
 	test-edit.sh \
 	test-escapes.sh \
 	test-events.sh \
+	test-file-attrs.sh \
 	test-find0.sh \
 	test-glob.sh \
 	test-inspect.sh \
-- 
1.8.5.3

